src/sentry/tasks/statistical_detectors.py

```diff
@@ -33,39 +34,39 @@ def run_detection() -> None:
 
     now = timezone.now()
 
-    performance_projects: List[int] = options.get(
-        "statistical_detectors.enable.projects.performance"
+    enabled_performance_projects: Set[int] = set(
+        options.get("statistical_detectors.enable.projects.performance")
+    )
+    enabled_profiling_projects: Set[int] = set(
+        options.get("statistical_detectors.enable.projects.profiling")
     )
-    profiling_projects: List[int] = options.get("statistical_detectors.enable.projects.profiling")
 
-    """ disabled for now so we can run experiements
+    performance_projects = []
+    profiling_projects = []
+
     for project in RangeQuerySetWrapper(
         Project.objects.filter(status=ObjectStatus.ACTIVE),
-        result_value_getter=lambda item: item.id,
-        step=ITERATOR_CHUNK,
+        step=100,
```

Comment:
- Is it safe to run this hourly?
- For the feature flag, I also need the organization, can I just pair this with a `.select_related("organization")`?

Reply:
> Is it safe to run this hourly?

Your loop body looks pretty fast as it is doing some light logic and queuing more tasks. You could use the `enabled_performance_projects` and `enabled_profiling_projects` lists to create an allow list of ids to reduce the time burnt iterating all of the projects. 

> For the feature flag, I also need the organization, can I just pair this with a .select_related("organization")?

Yes. You'll want to eager load the organization relation.

Reply:
So we're currently just iterating the two lists to avoid querying postgres as an experiment. As the next incremental step, we'd like to be able to test iterating all the projects and filtering it down via a feature flag. And eventually, the feature flag should go away, and we're just iterating all the projects.

But from your comment, it sounds like it should be okay to use this to iterate all the projects hourly and join it with the organization. I have seen other examples where it iterates all the projects (does not join with organizations) but runs every time 10 minutes. So i just wanted to double check